### PR TITLE
added staging changes

### PR DESCRIPTION
**Pull request by sauravk-webonise**
Added Staging Changes

### CODE REVIEW

File: webonise.txt [hunk 1]
```diff
@@ -50,3 +50,5 @@ int main() {
   return 0;
 }
 
+
+I am adding staging changes here --"staging changes"
```
Comment (sagarp-webonise) [inline webonise.txt]: modify staging changes to "something else" in same commit. do not add separate commit

Comment (sauravk-webonise) [reply webonise.txt:54]: Do need to change the commit message or the words inside the file..?
